packages/kit/src/runtime/server/page/render.js

```diff
@@ -73,6 +73,15 @@ export async function render_response({
 			: null;
 
 	if (page_config.ssr) {
+		if (!branch.at(-1)?.node.component) {
```

Comment:
Not wrapped with `DEV` because I noticed it's `false` when running `vite dev` in `sites/kit.svelte.dev`, similar to what [Rich encountered](https://github.com/sveltejs/kit/pull/8429#issuecomment-1378825746). I'm wondering if this hints at a bigger problem? Is `DEV` not working as expected after all?

Reply:
I filed an issue so we can have a single place to investigate: https://github.com/sveltejs/kit/issues/8492

Reply:
I sent a fix to Vite for the `DEV` bug